lib/fog/compute/xen_server/models.rb

```diff
@@ -2,80 +2,80 @@ module Fog
-        autoload :PifMetrics, 'fog/compute/xen_server/models/pif_metrics'
-        autoload :Pifs, 'fog/compute/xen_server/models/pifs'
-        autoload :PifsMetrics, 'fog/compute/xen_server/models/pifs_metrics'
-        autoload :Pool, 'fog/compute/xen_server/models/pool'
-        autoload :PoolPatch, 'fog/compute/xen_server/models/pool_patch'
-        autoload :PoolPatchs, 'fog/compute/xen_server/models/pool_patchs'
-        autoload :Pools, 'fog/compute/xen_server/models/pools'
-        autoload :Role, 'fog/compute/xen_server/models/role'
-        autoload :Roles, 'fog/compute/xen_server/models/roles'
-        autoload :Server, 'fog/compute/xen_server/models/server'
-        autoload :ServerAppliance, 'fog/compute/xen_server/models/server_appliance'
-        autoload :ServerAppliances, 'fog/compute/xen_server/models/server_appliances'
-        autoload :ServerMetrics, 'fog/compute/xen_server/models/server_metrics'
-        autoload :ServersMetrics, 'fog/compute/xen_server/models/servers_metrics'
-        autoload :Servers, 'fog/compute/xen_server/models/servers'
-        autoload :StorageManager, 'fog/compute/xen_server/models/storage_manager'
-        autoload :StorageManagers, 'fog/compute/xen_server/models/storage_managers'
-        autoload :StorageRepositories, 'fog/compute/xen_server/models/storage_repositories'
-        autoload :StorageRepository, 'fog/compute/xen_server/models/storage_repository'
-        autoload :Tunnel, 'fog/compute/xen_server/models/tunnel'
-        autoload :Tunnels, 'fog/compute/xen_server/models/tunnels'
-        autoload :Vbd, 'fog/compute/xen_server/models/vbd'
-        autoload :VbdMetrics, 'fog/compute/xen_server/models/vbd_metrics'
-        autoload :Vbds, 'fog/compute/xen_server/models/vbds'
-        autoload :VbdsMetrics, 'fog/compute/xen_server/models/vbds_metrics'
-        autoload :Vdi, 'fog/compute/xen_server/models/vdi'
-        autoload :Vdis, 'fog/compute/xen_server/models/vdis'
-        autoload :Vif, 'fog/compute/xen_server/models/vif'
-        autoload :VifMetrics, 'fog/compute/xen_server/models/vif_metrics'
-        autoload :Vifs, 'fog/compute/xen_server/models/vifs'
-        autoload :VifsMetrics, 'fog/compute/xen_server/models/vifs_metrics'
-        autoload :Vlan, 'fog/compute/xen_server/models/vlan'
-        autoload :Vlans, 'fog/compute/xen_server/models/vlans'
-        autoload :Vmpp, 'fog/compute/xen_server/models/vmpp'
-        autoload :Vmpps, 'fog/compute/xen_server/models/vmpps'
-        autoload :Vtpm, 'fog/compute/xen_server/models/vtpm'
-        autoload :Vtpms, 'fog/compute/xen_server/models/vtpms'
+        autoload :Blob, File.expand_path('../models/blob', __FILE__)
+        autoload :Blobs, File.expand_path('../models/blobs', __FILE__)
+        autoload :Bond, File.expand_path('../models/bond', __FILE__)
+        autoload :Bonds, File.expand_path('../models/bonds', __FILE__)
+        autoload :ClassMethods, File.expand_path('../models/class_methods', __FILE__)
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.